content/content-strategy/audit-content/topic-takeaway-listitems.md

```diff
@@ -0,0 +1,2 @@
+- A detailed view of your content
```

Comment:
Is this supposed to be in a callout partial? If so, it's missing the YAML front matter and may not render properly on the page. Likely to display file not found, 'topic-takeaway-listitems.md' to the left of the page container. cc @Phend-117